sql/table.h

```diff
@@ -1870,6 +1870,9 @@ enum enum_schema_table_state
 enum enum_fk_option { FK_OPTION_UNDEF, FK_OPTION_RESTRICT, FK_OPTION_NO_ACTION,
   FK_OPTION_CASCADE, FK_OPTION_SET_NULL, FK_OPTION_SET_DEFAULT };
 
+enum enum_fk_relation { FIELDS_NOT_EXIST, FIELDS_NULL,
+  FIELDS_NOT_NULL };
```

Comment:
These values had better be documented with `/** */` comments. What exactly do `FIELDS_NULL` and `FIELDS_NOT_NULL` mean if there are multiple columns and only some of them allow `NULL` values, some not?